### PR TITLE
fix what youll learn design[HSTN-746]

### PR DESCRIPTION
**Pull request by hngerebara**
Fix Styling
Before:
Screenshot 2019-11-27 at 3 17 23 PM

After:
Screenshot 2019-11-27 at 3 11 31 PM

### CODE REVIEW

File: assets/css/guides.less [hunk 1]
```diff
@@ -1091,6 +1091,17 @@ body .guides .guides-section-white {
       background: transparent !important;
     }
   }
+
+  h3#what-youll-learn-in-this-guide ~ .dlist {
```
Comment (brikis98) [inline assets/css/guides.less]: If you're already target an ID, why bother with the `h3`?

And what does the `~` selector do?

Also, should _all_ links in a definition list be styled this way? Or _only_ the ones in "what you'll learn in this guide" section?

Comment (hngerebara) [reply assets/css/guides.less]: Taken off the h3. 
~ is a sibling selector which I used to select `.dlist` class
Only what is within the `what you'll learn in this guide` section needs to be styled this way; that's why I'm selecting the id of that section

**Comment by hngerebara** [comment]
@brikis98 Please review once more

File: assets/css/guides.less (hunk 1 above)
Comment (brikis98) [reply assets/css/guides.less]: Roger

**Pull request merged**